oas_docs/bundle.json

```diff
@@ -2413,7 +2414,6 @@
                     "consumer",
                     "schedule",
                     "actions",
-                    "params",
```

Comment:
Why it get removed from the `required`? What is it because it has a `defaultValue`?

Reply:
I'd would think it's because of the default value, yes. You could try removing it and checking the bundle by yourself. Just follow the instructions in [x-pack/plugins/alerting/docs/openapi/README.md](x-pack/plugins/alerting/docs/openapi/README.md)

Reply:
Why did you add the defaultValue? Shouldn't it throw if it's missing in the request?